pset-iterators.js

```diff
@@ -182,7 +223,22 @@ Expected Result Shape: An array of objects (with each object representing a pres
 ************/
 console.log('Problem 7:')
 
+const previousPresidents = presidents.filter(president => {
+  if(president.left_office !== null){
+    return president
+  }
+})
 // Add your code below this line
+const shortTermPresidents = previousPresidents.filter(president => {
+  const tookOffice = parseInt(president.took_office.substring(0,4))
+  const leftOffice = parseInt(president.left_office.substring(0,4))
+
+  if(leftOffice - tookOffice < 4){
+    return president
+  }
+})
+
+console.log(shortTermPresidents)
```

Comment:
@nmikhol Good Job using the filter method for this solution. Good job using the parseInt to change the values of `took_office` and `left_office` to a number and then subtract the two values and see if they're less than 4 and return the president if true.

Take a look at this alternate solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L201)